pom.xml

```diff
@@ -90,6 +90,13 @@
             <version>3.11.1</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.behaviosec</groupId>
+            <artifactId>behaviosec-iSDK-2.0-SNAPSHOT</artifactId>
+            <version>2.0</version>
+            <scope>system</scope>
+            <systemPath>${project.basedir}/src/main/resources/behaviosec-iSDK-2.0-SNAPSHOT.jar</systemPath>
+        </dependency>
```

Comment:
SDK should be downloaded from a mvn repository (vs pointed to on the local file system. Otherwise customer won't be able to build.

Reply:
added information to request SDK download from support.